src/base-types.ts

```diff
@@ -6,6 +6,7 @@ export interface ContextParams {
   API_SECRET_KEY: string;
   SCOPES: string[] | AuthScopes;
   HOST_NAME: string;
+  HOST_SCHEME?: string;
```

Comment:
Do we want this to be optional? Or is there a separate type for input (where it is optional) and after applying defaults (where it will be set)?

Reply:
I'm making it optional to prevent the change from breaking. Otherwise existing users that update the package will get Typescript errors because the type-checker will expect that value to be passed.